packages/core-data/src/index.js

```diff
@@ -70,7 +70,7 @@ register( store );
 export { default as EntityProvider } from './entity-provider';
 export { default as useEntityRecord } from './hooks/use-entity-record';
 export { default as useEntityRecords } from './hooks/use-entity-records';
-export { default as __experimentalUseResourcePermissions } from './hooks/use-resource-permissions';
+export { default as useResourcePermissions } from './hooks/use-resource-permissions';
```

Comment:
It looks like a good opportunity to re-export those APIs directly from `./hooks` that is used to generate API docs.